src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/model/response/RestrictedPatientContent.kt

```diff
@@ -10,15 +10,25 @@ data class RestrictedPatientContent(
   @Schema(description = "The prisoner number (Nomis ID)", example = "A1234AA")
   val prisonerNumber: String,
   @Schema(
-    description = "Prison where the offender is supported by POM.  This is from AGENCY_LOCATIONS.AGY_LOC_ID in NOMIS.",
+    description = "Prison ID where the offender is supported by POM.  This is from AGENCY_LOCATIONS.AGY_LOC_ID in NOMIS.",
     example = "LEI",
   )
   val supportingPrisonId: String,
+  @Schema(
+    description = "Prison where the offender is supported by POM.",
+    example = "LEI",
```

Comment:
Minor but the example isn't a description

Reply:
thanks, will fix in next PR